changelog.txt

```diff
@@ -1,9 +1,31 @@
+4.30.0
+==========
+### New Features:
+**Added support for Proof Key for Code Exchange (PKCE) in confidential client authorization code flows**. See issue [#1473](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1473).  
+
+### Enhancements:
+**Improved the error message when an app is unable to listen to system browser on localhost URL for interactive flow**. See issue [#2219](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2219).  
+**MSAL.NET will now fall back to WebView1 if WebView2 is unavailable** on .NET 5, .NET Core, and .NET classic. See issue [#2495](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2495).  
+**MSAL.NET now, by default, enables a partitioned token serialization cache for client credential flow**. See issue [#2544](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2544).  
```

Comment:
Performance enhancement?

Reply:
What is the suggestion? Mention that ^ in the description?